app-misc/mosquitto/patches/mosquitto-2.0.14.patchset

```diff
@@ -0,0 +1,22 @@
+From d74683d456a94b57a806db50824b34391375e1b9 Mon Sep 17 00:00:00 2001
+From: Al Hoang <3811822-hoanga@users.noreply.gitlab.com>
+Date: Sat, 16 Apr 2022 11:32:48 -0500
+Subject: Haiku doesn't use -ldl, add -lnetwork to CMakeList.txt
+
+
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 9380a04..f95760d 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -179,6 +179,8 @@ if (UNIX)
+ 	    set (MOSQ_LIBS ${MOSQ_LIBS} m network)
+     elseif(QNX)
+         set(MOSQ_LIBS ${MOSQ_LIBS} m socket)
++    elseif (${CMAKE_SYSTEM_NAME} MATCHES "Haiku")
++	    set (MOSQ_LIBS ${MOSQ_LIBS} m network)
+     else(APPLE)
+         set (MOSQ_LIBS ${MOSQ_LIBS} dl m)
+         find_library(LIBRT rt)
+-- 
+2.30.2
+
```

Comment:
Actually the patch isn't needed anymore, seems upstreamed :)

Reply:
less patches sounds good to me! thought the patch was needed but looks like not so. i have pushed an update without the patch and looks like it is building from my light testing so far.

and apologies on the earlier patch mangling the author line, i think i  missed the (magical) incantation to update the patch + preserve original author option